packages/@dcl/inspector/src/components/EntityInspector/EntityInspector.tsx

```diff
@@ -1,19 +1,32 @@
-import { useMemo } from 'react'
+import { useCallback, useMemo } from 'react'
+import { Entity } from '@dcl/ecs'
 
 import { useEntitiesWith } from '../../hooks/sdk/useEntitiesWith'
 import { TransformInspector } from './TransformInspector'
 import { SceneInspector } from './SceneInspector'
 import { ROOT } from '../../lib/sdk/tree'
+import { useSdk } from '../../hooks/sdk/useSdk'
+import { isLastWriteWinComponent } from '../../hooks/sdk/useComponentValue'
 
 export const EntityInspector: React.FC = () => {
   const selectedEntities = useEntitiesWith((components) => components.EntitySelected)
+  const sdk = useSdk()
   const entity = useMemo(() => (selectedEntities.length > 0 ? selectedEntities[0] : null), [selectedEntities])
 
+  const handleRemove = (entity: Entity, componentId: number) => {
```

Comment:
why is this piece of logic here instead of within the `TransformInspector`? To me it seems an unnecesary coupling of the parent and the children, constructing this here and passing it as a prop, instead of simply doing a `Transform.deleteFrom(entity)` inside the component.